src/main/java/io/boomerang/service/EventProcessorImpl.java

```diff
@@ -199,7 +204,7 @@ private EventResponse processEvent(CloudEvent<AttributesImpl, JsonNode> event, S
       property.setKey("eventId");
       property.setValue(event.getAttributes().getId());
       cloudEventLabels.add(property);
-      executionRequest.setLabels(cloudEventLabels);
+      // executionRequest.setLabels(cloudEventLabels);
```

Comment:
@amhudson how does not setting the labels on a CloudEvent (listener triggered) execution affect the webhook?